sphinx/directives/other.py

```diff
@@ -383,6 +472,7 @@ def setup(app: "Sphinx") -> Dict[str, Any]:
     directives.register_directive('hlist', HList)
     directives.register_directive('only', Only)
     directives.register_directive('include', Include)
+    directives.register_directive('includedoc', IncludeDocument)
```

Comment:
This name is bad, would appreciate feedback on how to handle this.

Reply:
Alternative directives specific to the use case: `parse`, `includeother`, `includeparsed`, etc.